torchaudio/prototype/pipelines/hifigan_pipeline.py

```diff
@@ -0,0 +1,125 @@
+
+
+@dataclass
+class HiFiGANGeneratorBundle:
+    """Data class that bundles associated information to use pretrained
+    :py:class:`~torchaudio.prototype.models.HiFiGANGenerator`.
+
+    This class provides interfaces for instantiating the pretrained model along with
+    the information necessary to retrieve pretrained weights and additional data
+    to be used with the model.
+
+    Torchaudio library instantiates objects of this class, each of which represents
+    a different pretrained model. Client code should access pretrained models via these
+    instances.
+
+    Please see below for the usage example.
+
+    Example: Synthetic spectrogram to audio.
+        >>> import torch
+        >>> import torchaudio
+        >>> # Since HiFiGAN bundle is in prototypes, it needs to be exported explicitly
+        >>> from torchaudio.prototype.pipelines import HIFIGAN_GENERATOR_V3_LJSPEECH as bundle
+        >>>
+        >>> # Load the HiFiGAN bundle
+        >>> vocoder = bundle.get_vocoder()
+        Downloading: "https://download.pytorch.org/torchaudio/models/hifigan_generator_v3_ljspeech.pth"
+        100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5.59M/5.59M [00:00<00:00, 18.7MB/s]
+        >>>
+        >>> # Generate synthetic mel spectrogram
+        >>> specgram = torch.sin(0.5 * torch.arange(start=0, end=100)).expand(bundle._params["in_channels"], 100)
+        >>>
+        >>> # Trasform mel spectrogram into audio
+        >>> waveforms = vocoder(specgram)
+        >>> torchaudio.save('sample.wav', waveforms, bundle.sample_rate)
+
+
+    Example: Usage together with Tactron2, text to audio.
+        >>> import torch
+        >>> import torchaudio
+        >>> # Since HiFiGAN bundle is in prototypes, it needs to be exported explicitly
+        >>> from torchaudio.prototype.pipelines import HIFIGAN_GENERATOR_V3_LJSPEECH as bundle_hifigan
+        >>>
+        >>> # Load Tactron2 bundle
+        >>> bundle_tactron2 = torchaudio.pipelines.TACOTRON2_WAVERNN_CHAR_LJSPEECH
+        >>> processor = bundle_tactron2.get_text_processor()
+        >>> tacotron2 = bundle_tactron2.get_tacotron2()
+        >>>
+        >>> # Use Tactron2 to convert text to mel spectrogram
+        >>> text = "A quick brown fox jumped over a lazy dog"
+        >>> input, lengths = processor(text)
+        >>> specgram, lengths, _ = tacotron2.infer(input, lengths)
+        >>>
+        >>> # Load HiFiGAN bundle
+        >>> vocoder = bundle_hifigan.get_vocoder()
+        Downloading: "https://download.pytorch.org/torchaudio/models/hifigan_generator_v3_ljspeech.pth"
+        100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5.59M/5.59M [00:03<00:00, 1.55MB/s]
```

Comment:
Can you remove redundant `█`, so that it fits the 120 line limit? (I think 80 chars is enough)

Reply:
Done